### PR TITLE
New blog post: Good First Issues list

### PR DESCRIPTION
**Pull request by hyles-lineata**
Title seems bland-- feel free to improve [anything] if inspiration strikes.

### CODE REVIEW

File: _posts/2019-11-08-first-issues-list-11-2019.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Our [issue list on GitHub](https://github.com/bokeh/bokeh/issues) is a good place to start to see what both the project maintainers and the community consider to be important or useful opportunities for improvement. Several of these issues are tagged [**good first issue**](https://github.com/bokeh/bokeh/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), which are some of the more accessible fixes and improvements, and a great way for a new contributor to make a difference. Good first issues generally come in two flavors: technical, but specific and contained; and less technical but broad, touching several aspects of the project to accomplish a goal.
+
+We’ve chosen some of these good first issues to highlight, and encourage our users to get involved. For all of these, contributors should feel free to add to the discussion of the issue and get a clear picture of how best to solve the problem— if you have a question about an issue, chances are very good that other people have the same question! We’d like to make this a regular feature on the Bokeh blog, to encourage new contributors on an ongoing basis. Here’s our inaugural list.
```
Comment (bryevdv) [inline _posts/2019-11-08-first-issues-list-11-2019.md]: Suggest proper `&mdash;` or full stop and new sentence after "problem". Also I think it makes it easier to comment on drafts in the future if the text is either:
* formatted to a specific line length, or 
* has one sentence per line

File: _posts/2019-11-08-first-issues-list-11-2019.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+featured_image:
+excerpt: >
+  Our first post of a regular feature focusing on entry points for new contributors.
+draft: false
+
+---
+
+We’re always excited when Bokeh users express interest in becoming contributors, and usually the first question is: how can I help?
+
+Our [issue list on GitHub](https://github.com/bokeh/bokeh/issues) is a good place to start to see what both the project maintainers and the community consider to be important or useful opportunities for improvement. Several of these issues are tagged [**good first issue**](https://github.com/bokeh/bokeh/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), which are some of the more accessible fixes and improvements, and a great way for a new contributor to make a difference. Good first issues generally come in two flavors: technical, but specific and contained; and less technical but broad, touching several aspects of the project to accomplish a goal.
```
Comment (bryevdv) [inline _posts/2019-11-08-first-issues-list-11-2019.md]: maybe (paraphrase suggestion) "good place to start... but can be daunting if you new to the project. To make things easier for you, several of these...."

**Comment by bryevdv** [comment]
few small comments otherwise LGTM

File: _posts/2019-11-08-first-issues-list-11-2019.md (hunk 1 above)
Comment (hyles-lineata) [reply _posts/2019-11-08-first-issues-list-11-2019.md]: Re-punctuated, and noted.

**Pull request merged**